### PR TITLE
Throttle: allow removal of throttler buckets

### PR DESCRIPTION
**Pull request by element-code**
This allows to manual remove throttler buckets manually.
E.g. after paying a plan or a successful login.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPdocs
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Throttle/Throttler.php [hunk 1]
```diff
@@ -151,6 +151,14 @@ public function check(string $key, int $capacity, int $seconds, int $cost = 1):
 		return false;
 	}
 
+	public function remove(string $key) : void
```
Comment (paulbalandan) [inline system/Throttle/Throttler.php:154]:
```suggestion
	public function remove(string $key): void
```

**Review by paulbalandan — CHANGES_REQUESTED** [review]
This needs some tests and user guide mention.

**Comment by lonnieezell** [comment]
@element-code are you able to add tests to this?

**Comment by element-code** [comment]
Yes, but it will take some time, see https://github.com/codeigniter4/CodeIgniter4/pull/3911#issuecomment-740486515
Help is appreciated :)

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**